src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/CustomAttributeBuilder.Mono.cs

```diff
@@ -184,17 +184,18 @@ private static bool IsValidValue(Type type, object value)
             if (fieldValues == null)
                 throw new ArgumentNullException(nameof(fieldValues));
             if (con.GetParametersCount() != constructorArgs.Length)
-                throw new ArgumentException("Parameter count does not match " +
-                        "passed in argument value count.");
+                throw new ArgumentException(SR.Argument_BadParameterCountsForConstructor);
             if (namedProperties.Length != propertyValues.Length)
-                throw new ArgumentException("Array lengths must be the same.",
-                        "namedProperties, propertyValues");
+                throw new ArgumentException(SR.Arg_ArrayLengthsDiffer, "namedProperties, propertyValues");
```

Comment:
Pre-existing, but I've not seen us pass two parameter names to an argument exception.  Is that a standard thing?

Reply:
Looking for `new ArgumentException\([^\n]*"[^\n"]+,[^"\n]+"` I see a few other hits in our code. It's not pretty but if you're diagnosing something, it's probably welcome?